chat-plugins/info.js

```diff
@@ -2614,18 +2616,35 @@ var commands = exports.commands = {
 		if (!this.can('declare', null, room)) return false;
 		if (!this.canBroadcast()) return;
 		if (this.room.isPersonal && !this.user.can('announce')) {
-			this.errorReply("Images are not allowed in personal rooms.");
-			return false;
+			return this.errorReply("Images are not allowed in personal rooms.");
 		}
 
 		var targets = target.split(',');
-		if (targets.length !== 3) {
-			return this.parse('/help showimage');
-		}
+		if (targets.length !== 3) return this.parse('/help showimage');
+
+		var image = targets[0].trim();
+		if (!image) return this.errorReply('No image URL was provided!');
+
+		var imageUrl = url.parse(image);
```

Comment:
That's not the main reason I'm using it; I'm using it to escape `"`, `/` `>`, and `<` so users can't add extra HTML.